<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewClass</class>
 <widget class="QMainWindow" name="AddNewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>525</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CLASS ID:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>621</width>
      <height>331</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>#</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Student ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>First name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Gender</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Birth date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Social ID</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="classlineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateStudentBtt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeStudentBtt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addStudentBtt">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+ Add Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadFileBtt">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtt">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
